tests/ludwig/datasets/model_configs/train_all_model_configs.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
+#
+# Trains a ludwig model for every dataset which has a default_model_config.
+# You must have valid kaggle credentials in your environment, a few GB of disk space, and good internet bandwidth.
+# Also, for each dataset associated with a Kaggle competition you'll need to sign in to Kaggle and accept the terms of
+# the competition.
+#
+import multiprocessing
+import time
+from dataclasses import dataclass
+from typing import Optional
+
+import pandas as pd
+
+from ludwig import datasets, visualize
+from ludwig.api import LudwigModel
+
+
+@dataclass
+class TrainingResults:
```

Comment:
We can also add the current commit hash of the repo, as performance could change based on that.

Reply:
Done.